bindings/wasm/examples-account/src/create_vp.ts

```diff
@@ -12,58 +13,125 @@ import {
     Resolver,
     SubjectHolderRelationship,
     Timestamp,
-    VerifierOptions
-} from '@iota/identity-wasm';
-import {createVC} from './create_vc';
+    VerifierOptions,
+    Storage,
+    MethodContent,
+    Network
+} from './../../node/identity_wasm.js';
 
 /**
  This example shows how to create a Verifiable Presentation and validate it.
  A Verifiable Presentation is the format in which a (collection of) Verifiable Credential(s) gets shared.
  It is signed by the subject, to prove control over the Verifiable Credential with a nonce or timestamp.
-
- @param {{network: Network, explorer: ExplorerUrl}} clientConfig
  **/
-async function createVP(clientConfig) {
-    // An issuer creates and signs a Verifiable Credential and passes it to the credential subject `Alice`.
+async function createVP(storage?: Storage) {
+    // ===========================================================================
+    // Step1: Create identities for the issuer and the holder.
```

Comment:
Nit: formatting. Applies to all `StepN` comments.

```suggestion
    // Step 1: Create identities for the issuer and the holder.
```